Source/WebCore/platform/graphics/skia/PlatformDisplaySkia.cpp

```diff
@@ -39,8 +39,35 @@ IGNORE_CLANG_WARNINGS_END
 #include <skia/gpu/gl/GrGLInterface.h>
 #include <skia/gpu/gl/GrGLTypes.h>
 
```

Comment:
Why this empty line? We normally use a single block of includes

Reply:
No reason, it's just the style checker didn't catch it. I will remove it before landing.